client/src/app/+login/login.component.html

```diff
@@ -107,7 +111,7 @@ <h4 i18n class="modal-title">Forgot your password</h4>
     >
 
     <input
-      type="submit" i18n-value value="Send me an email to reset my password" class="action-button-submit"
+      type="submit" i18n-value value="Reset" class="action-button-submit"
```

Comment:
```suggestion
      type="submit" i18n-value="Password reset button" value="Reset" class="action-button-submit"
```

Reply:
Maybe the opposite ? `i18n-value="Reset" value="Password reset button"`

Reply:
@rigelk the `i18n-value` is the identifier ?

Reply:
@kimsible please refer to the relevant angular documentation: https://angular.io/guide/i18n#mark-element-attributes-for-translations